sdk/python/src/freewayai/prompt_formatted.py

```diff
@@ -1,9 +1,40 @@
+import copy, json
 from .prompt_messages_to_provider_format import convert_messages_to_format
+from .token_counting import count_messages_tokens
+from .utilities import _format_messages, _format_template_string
```

Comment:
Same with these. I think you're trying to say that all of your utility functions are private, so marking them with `_`, but sort of all of this code is private. We're exposing just the two classes in the `__all__.py` file and the usage is `from freewayai import LLMSystem`. I think it's safe to assume any other methods for our internal implementation are all equally private.